File: functions/notes.py
# Governor-Nephi Bakare, Functions Notes Python

# functions hold/organize actions to be reused


#def add(numOne, numTwo):#parameters set the name of the variables (just for the function)

    
    #numOne = int(input("Please tell me a number: \n"))
    #numTwo = int(input("Please tell me a number: \n"))
    #print (numOne+numTwo)
    #add(int(input("Please tell me a number: \n")))
    #dd()

#number = int(input("Please tell me a number: \n"))

#print(add(number,21))

def values(type):
    return input(f"Please give me a {type}:\n")


name = values("names")
place = values("place")
verb = values("verb")

print(f"{name} was realy fast getting to {place} because they {verb} the whole way there.")

File: functions/notes.c
// Governor-Nephi Bakare, Funtions Notes C
#include <stdio.h>

int num;
char name [1000], verb[1000];
void add(int numOne, int numTwo){
    //printf("%d\n", numOne + numTwo);
    //return numOne+numTwo;
}
void due(char assignment[75], char day[30]){
    printf("The %s assignment is due %s.\n", assignment, day);
}

int main(void){
    //printf("Please tell me a number:\n");
    //scanf("%d", num);
    //add(num, 10);
    //add(4,10);
    //add(4,10);
    //add(4,10);
    due("Functions Notes", "Today");
    due("Hello World Update", "Tomorrow");
    due("Financial Calculator", "Friday");
    return 0;
}